app/controllers/LookupController.scala

```diff
@@ -63,30 +65,44 @@ class LookupController @Inject() (
   val lookupResultID  = "psa-lookup-result"
 
   def displayNotFoundResults: Action[AnyContent] = actionWithSessionId.async { implicit request =>
-    sessionCacheService
-      .fetchAndGetFormData[PSALookupRequest](lookupRequestID)
-      .flatMap {
-        case Some(req @ PSALookupRequest(_, Some(_))) =>
-          Future.successful(Ok(psa_lookup_not_found_results(req, buildTimestamp)))
-        case _ =>
-          Future.successful(
-            Redirect(routes.LookupSchemeAdministratorReferenceController.displaySchemeAdministratorReferenceForm)
-          )
-      }(executionContext)
+    if (appConfig.psalookupjourneyShutterEnabled) {
+      Future.successful(Ok(withdrawnPSALookupJourney()))
+    } else {
+      sessionCacheService
+        .fetchAndGetFormData[PSALookupRequest](lookupRequestID)
+        .flatMap {
+          case Some(req @ PSALookupRequest(_, Some(_))) =>
+            Future.successful(Ok(psa_lookup_not_found_results(req, buildTimestamp)))
+          case _ =>
+            Future.successful(
+              Redirect(routes.LookupSchemeAdministratorReferenceController.displaySchemeAdministratorReferenceForm)
+            )
+        }(executionContext)
+    }
   }
 
   def displayLookupResults: Action[AnyContent] = actionWithSessionId.async { implicit request =>
-    sessionCacheService
-      .fetchAndGetFormData[PSALookupResult](lookupResultID)
-      .map {
-        case Some(result) => Ok(psa_lookup_results(result, buildTimestamp))
-        case None =>
-          Redirect(routes.LookupSchemeAdministratorReferenceController.displaySchemeAdministratorReferenceForm)
-      }(executionContext)
+    if (appConfig.psalookupjourneyShutterEnabled) {
+      Future.successful(Ok(withdrawnPSALookupJourney()))
+    } else {
+      sessionCacheService
+        .fetchAndGetFormData[PSALookupResult](lookupResultID)
+        .map {
+          case Some(result) => Ok(psa_lookup_results(result, buildTimestamp))
+          case None =>
+            Redirect(routes.LookupSchemeAdministratorReferenceController.displaySchemeAdministratorReferenceForm)
+        }(executionContext)
+    }
   }
 
   def displayProtectionTypeGuidance: Action[AnyContent] =
-    actionWithSessionId(implicit request => Ok(pla_protection_guidance()))
+    actionWithSessionId { implicit request =>
+      if (appConfig.psalookupjourneyShutterEnabled) {
+        Ok(withdrawnPSALookupJourney())
+      } else {
+        Ok(pla_protection_guidance())
+      }
+    }
 
   def redirectToStart: Action[AnyContent] = actionWithSessionId.async { implicit request =>
```

Comment:
I believe this method could also take the feature toggle into consideration. It is going to make it consitent with other methods and also is going to prevent unnecessary session removal.